readthedocs/projects/exceptions.py

```diff
@@ -15,6 +15,12 @@ class ProjectConfigurationError(BuildEnvironmentError):
         'Make sure you have a conf.py file in your repository.'
     )
 
+    MULTIPLE_CONF_FILES = _(
+        "There are more than one conf.py file and none of them say doc "
+        "in their path, we don't know which one use. Please, select "
+        "the correct one under the Advanced settings tab in the "
+        "project's Admin."
```

Comment:
We should probably provide a link here.

Reply:
Do we support HTML content on the message exception? I think no.

For example, we are not using `| safe` filter at https://github.com/rtfd/readthedocs.org/blob/bb7bb4d6751ecce67ba274a32100406b951f9b90/readthedocs/templates/builds/build_detail.html#L156

Reply:
Would make for a better UX, perhaps as an idea in the future. We could also add it as a "build tip"

Reply:
I like that idea. I think we need to firm up our usage around build tips and not do the logic in the template though.